test/graphql/plug/graphiql_test.exs

```diff
@@ -0,0 +1,146 @@
+defmodule GraphQL.Plug.GraphiQLTest do
```

Comment:
This seems to duplicate the endpoint tests, can we strip this down to just graphiql specific tests?